### ISSUE TITLE
Unable to install version 4.0

### ISSUE THREAD

**Issue by dfeyer** [truqu/elm-oauth2#13]
```
/elm install truqu/elm-oauth2                                                                                                         ✘ 1
Here is my plan:

  Add:
    truqu/elm-base64    2.0.4
    truqu/elm-oauth2    4.0.0

Would you like me to update your elm.json accordingly? [Y/n]: y
Starting downloads...

  ✗ truqu/elm-oauth2 4.0.0

-- HTTP PROBLEM ----------------------------------------------------------------

The following HTTP request failed:

    <https://github.com/truqu/elm-oauth2/zipball/4.0.0/>

Here is the error message I was able to extract:

    HttpExceptionRequest Request { host = "codeload.github.com" port = 443
    secure = True requestHeaders =
    [("Cookie","logged_in=no"),("User-Agent","elm/0.19.0"),("Accept-Encoding","gzip")]
    path = "/truqu/elm-oauth2/legacy.zip/4.0.0" queryString = "" method = "GET"
    proxy = Nothing rawBody = False redirectCount = 10 responseTimeout =
    ResponseTimeoutDefault requestVersion = HTTP/1.1 } (StatusCodeException
    (Response {responseStatus = Status {statusCode = 404, statusMessage = "Not
    Found"}, responseVersion = HTTP/1.1, responseHeaders =
    [("Content-Length","15"),("Access-Control-Allow-Origin","https://render.githubusercontent.com"),("Content-Security-Policy","default-src
    'none'; style-src 'unsafe-inline';
    sandbox"),("Strict-Transport-Security","max-age=31536000"),("Vary","Authorization,Accept-Encoding"),("X-Content-Type-Options","nosniff"),("X-Frame-Options","deny"),("X-XSS-Protection","1;
    mode=block"),("Date","Sat, 08 Sep 2018 11:59:25
    GMT"),("X-GitHub-Request-Id","E6CC:3685:3A5F44:794A6E:5B93B99D")],
    responseBody = (), responseCookieJar = CJ {expose = [Cookie {cookie_name =
    "logged_in", cookie_value = "no", cookie_expiry_time = 3018-01-09 00:00:00
    UTC, cookie_domain = "github.com", cookie_path = "/", cookie_creation_time =
    2018-09-08 11:59:25.167575202 UTC, cookie_last_access_time = 2018-09-08
    11:59:25.167716784 UTC, cookie_persistent = False, cookie_host_only = False,
    cookie_secure_only = True, cookie_http_only = True},Cookie {cookie_name =
    "_gh_sess", cookie_value =
    "dTNRUFJ3Zi9jemtzRHo4d0FtRm8raGRpU1J1SjZHbklxM3dXQWhOazkxN2xXVXBJK2U1UDF3cXZPSkFNazlBZWprNE8xNjRmZlpkSVJ1c0RsOGhpYmJ1L2JvZXU1TzI5SGQ1bGw2cmE5c0xkRDlPT25FS0JoTEVMRGQ5b1E0K1VKUjJjSmt0Y2ZLd29CcHhPZEIrWjhQRHYvaC8zTVE4YXlUdUQ0UE81VjlNTmgxdXhCcDVkSlBVZzVUclJGQmxlRHFHcHVJdkwyVTkzcnoyL0VnRVlkZz09LS1EQ0xRTUY3VXBjMnM5WWo4TloydG1nPT0%3D--098a9e1b85bab3cd35eb8d83c49f29cf896c9f38",
    cookie_expiry_time = 3018-01-09 00:00:00 UTC, cookie_domain = "github.com",
    cookie_path = "/", cookie_creation_time = 2018-09-08 11:59:25.167575202 UTC,
    cookie_last_access_time = 2018-09-08 11:59:25.167575202 UTC,
    cookie_persistent = False, cookie_host_only = True, cookie_secure_only =
    True, cookie_http_only = True},Cookie {cookie_name = "has_recent_activity",
    cookie_value = "1", cookie_expiry_time = 3018-01-09 00:00:00 UTC,
    cookie_domain = "github.com", cookie_path = "/", cookie_creation_time =
    2018-09-08 11:59:25.167575202 UTC, cookie_last_access_time = 2018-09-08
    11:59:25.167575202 UTC, cookie_persistent = False, cookie_host_only = True,
    cookie_secure_only = False, cookie_http_only = False}]}, responseClose' =
    ResponseClose}) "404: Not Found\n")
```

Look like a missing tag for 4.0.0 ?

**Comment by KtorZ** [comment]
Hmmmm... tag is indeed missing. Let me fix that.

**edit**
Fixed!

```
$ elm install truqu/elm-oauth2
Here is my plan:
  
  Add:
    elm/http            1.0.0
    elm/regex           1.0.0
    truqu/elm-base64    2.0.4
    truqu/elm-oauth2    4.0.0

Would you like me to update your elm.json accordingly? [Y/n]: Y
Starting downloads...

  ● truqu/elm-oauth2 4.0.0

Dependencies ready! 
```

**Comment by dfeyer** [comment]
Awesome response time, thanks a lots for this package and your dedication.

Bests

**Comment by KtorZ** [comment]
Thanks.
i am waiting for the tests to be available on 0.19 and will add some.

Meanwhile, if you find any quirks in the docs or in an existing authorisation provider implementation (like Facebook, Github and LinkedIn which have broken implementations _sigh_) , please do no hesitate to report them.

PR are also welcome :)
